packages/ui-tabs/src/Tabs/README.md

```diff
@@ -161,44 +341,134 @@ Set the height of the Tabs component with the `fixHeight` property (set to '100%
+    render() {
+      const { selectedIndex, heightOption } = this.state
+      const { heightOptions } = this
+
+      const containerProps = {
+        as: 'div',
+        ...(heightOption.includes('fixHeight') && {
+          height: '22rem',
+          withVisualDebug: true
+        })
+      }
+
+      return (
+        <>
+          <View display="block" margin="none none medium">
+            <RadioInputGroup
+              name="tabsHeightOptions"
+              defaultValue="fixHeight: 100%"
+              description={
+                <ScreenReaderContent>Tabs height selector</ScreenReaderContent>
+              }
+              variant="toggle"
+              onChange={this.handleHeightOptionSelect}
+            >
+              {Object.keys(heightOptions).map((heightOption) => (
+                <RadioInput
+                  key={heightOption}
+                  label={heightOption}
+                  value={heightOption}
+                />
+              ))}
+            </RadioInputGroup>
+          </View>
+
+          <View {...containerProps}>
+            <Tabs
+              margin="large auto"
+              padding="medium"
+              onRequestTabChange={this.handleTabChange}
+              {...heightOptions[heightOption]}
+            >
+              <Tabs.Panel
+                id="tabA"
+                renderTitle="Tab A"
+                textAlign="center"
+                padding="large"
+                iSelected={selectedIndex === 0}
+              >
+                <Button>Focus Me</Button>
+              </Tabs.Panel>
+              <Tabs.Panel id="tabB" renderTitle="Disabled Tab" isDisabled>
+                {lorem.paragraphs()}
+              </Tabs.Panel>
+              <Tabs.Panel
+                id="tabC"
+                renderTitle="Tab C"
+                isSelected={selectedIndex === 2}
+              >
+                {lorem.paragraphs()}
+              </Tabs.Panel>
+              <Tabs.Panel
+                id="tabD"
+                renderTitle="Tab D"
+                isSelected={selectedIndex === 3}
+              >
+                {lorem.paragraphs()}
+              </Tabs.Panel>
+            </Tabs>
+          </View>
+        </>
+      )
     }
+  }
 
-    this.heightOptions = {
+  render(<Example />)
+  ```
+
+- ```js
+  const Example = () => {
+    const [selectedIndex, setSelectedIndex] = useState(0)
+    const [heightOption, setHeightOption] = useState('fixHeight: 100%')
+
+    const heightOptions = {
       ['fixHeight: 100%']: { fixHeight: '100%' },
       ['fixHeight: 15rem']: { fixHeight: '15rem' },
       ['minHeight: 17rem']: { minHeight: '17rem' },
       ['maxHeight: 10rem']: { maxHeight: '10rem' }
     }
-  }
 
-  handleTabChange = (event, { index, id }) => {
-    this.setState({
-      selectedIndex: index
-    })
-  }
-
-  handleHeightOptionSelect = (e, heightOption) => {
-    this.setState({ heightOption })
-  }
+    const handleTabChange = (event, { index, id }) => {
```

Comment:
I think the 'id' should be removed from the rest of the code in the handleTabChange functions too to be consistent.

Reply:
Done